CHANGELOG.rst

```diff
@@ -1,5 +1,38 @@
 .. _changelog:
 
+0.37.0
+------
+
+Renku ``0.37.0`` introduces new Project Creation and updated Project Update API.
```

Comment:
How about ... improves the Knowledge Graph API, with a new Project Creation functionality and a Project Update enhancement.